PowerShellEditorServices.build.ps1

```diff
@@ -303,10 +320,6 @@ task LayoutModule -After Build {
     # Copy Third Party Notices.txt to module folder
     Copy-Item -Force -Path "$PSScriptRoot\Third Party Notices.txt" -Destination $psesOutputPath
 
-    # Copy UnixConsoleEcho native libraries
-    Copy-Item -Path "$script:PsesOutput/runtimes/osx-64/native/*" -Destination $psesDepsPath -Force
-    Copy-Item -Path "$script:PsesOutput/runtimes/linux-64/native/*" -Destination $psesDepsPath -Force
```

Comment:
this doesn't seem to be needed anymore... which is kinda odd to me... but moving to a newer SDK seems to not generate these folders anymore.

Reply:
I've had errors about those folders being missing since they were added tbh.  I assumed they were important to CI but I've just been commenting them out.

If the `UnixConsoleEcho` dlls still end up where they need to be then 🤷 probably remove it